exporter/clickhousemetricsexporter/clickhouse.go

```diff
@@ -100,7 +100,8 @@ func NewClickHouse(params *ClickHouseParams) (base.Storage, error) {
 		)
 		ENGINE = MergeTree
 			PARTITION BY toDate(timestamp_ms / 1000)
-			ORDER BY (metric_name, fingerprint, timestamp_ms);`, database, SAMPLES_TABLE, CLUSTER))
+			ORDER BY (metric_name, fingerprint, timestamp_ms)
+			TTL toDateTime(timestamp_ms) + toIntervalSecond(2592000);`, database, SAMPLES_TABLE, CLUSTER))
```

Comment:
Does this work correcly? Since the timestamp is in epoch millis?

Reply:
Good catch, let me update it.

Reply:
Updated @srikanthccv